src/styles/styles.scss

```diff
@@ -162,16 +162,24 @@ a {
 }
 
 .progress-bar {
-  display: inline;
+  display: inline-block;
   width: 100px;
-  background-color: $turing-yellow;
-  margin-left: 15px;
+  height: 28px;
+  background-color: var(--s-color-yellow-500);
+  margin-left: 20px;
+  padding-left: 10px;
```

Comment:
Can the borders be rounded? So it matches the rest of the look/feel?

Reply:
Yep! Great suggestion.